sgkit/io/vcf/vcf_reader.py

```diff
@@ -225,6 +241,17 @@ def vcf_to_zarrs(
         Width (number of samples) to use when storing chunks in output, by default 1,000.
     output_storage_options
         Any additional parameters for the storage backend, for the output (see ``fsspec.open``).
+    ploidy : int
```

Comment:
No need to add types here any more, since they are automatically generated. (Several of these below too.)